src/component/vocabulary/VocabularyManagement.tsx

```diff
@@ -38,6 +40,7 @@ export const VocabularyManagement: React.FC<VocabularyManagementProps> = props =
     </>
 
     return <div>
+        <Helmet><title>{`${i18n("vocabulary.management.vocabularies")} | ${Constants.APP_NAME}`}</title></Helmet>
```

Comment:
Most of them seem to be extractable to a separate component.

Reply:
Good point, done.